config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml

```
@@ -445,6 +445,52 @@ presubmits:
             privileged: true
 
 periodics:
+- interval: 168h # one week
+  name: nodeoutofservice-volumedetach-e2e-gce-cloud-provider
+  cluster: k8s-infra-prow-build
+  labels:
+    preset-service-account: "true"
+    preset-k8s-ssh: "true"
+    preset-use-new-registry: "true"
+    preset-dind-enabled: "true"
+    preset-pull-kubernetes-e2e: "true"
+    preset-pull-kubernetes-e2e-gce: "true"
+  spec:
+    containers:
+    - args:
+      - --timeout=100
+      - --bare
+      - --scenario=kubernetes_e2e
+      - --
+      - --check-leaked-resources
+      - --env=KUBE_FEATURE_GATES=NodeOutOfServiceVolumeDetach=true
+      - --extract=ci/latest-fast
+      - --extract-ci-bucket=k8s-release-dev
+      - --gcp-master-image=gci
+      - --gcp-node-image=gci
+      - --gcp-nodes=4
+      - --gcp-zone=us-west1-b
+      - --ginkgo-parallel=30
+      - --provider=gce
+      - --test_args=--ginkgo.focus=\[Feature:(NodeOutOfServiceVolumeDetach)\]
+      - --timeout=50m
+      image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20220727-f055b40439-master
+      resources:
+        limits:
+          cpu: 2
+          memory: 6Gi
+        requests:
+          cpu: 2
+          memory: 6Gi
+      securityContext:
+        privileged: true
+  annotations:
+    testgrid-dashboards: google-gce, google-gci, provider-gcp-periodics
+    testgrid-tab-name: nongraceful-node-shutdown
+    testgrid-num-failures-to-alert: '6'
+    testgrid-alert-email: release-team@kubernetes.io
```

```suggestion
    testgrid-alert-email: release-team@kubernetes.io
```

Why the release team need to informed in case of failure ?